Gallery/styles.css

```diff
@@ -0,0 +1,187 @@
+:root {
+  --red: #e20000;
+  --white: #ffffff;
+}
+
+
+html {
+  scroll-behavior: smooth;
+}
+
+
+
+
+img{
+  height: 100vh;
+  width: 100vw;
+  position: static;
+}
+
+nav{
+  width: 100vw;
+  position: sticky;
+  color: white;
+  background-color: cyan;
+  padding-top: 0.5em;
+  font-weight: 300;
+  padding-left: 10px;
+  font-size: 1.2em;
+}
+
+
+ul{
+  display: flex;
+  flex-direction: row;
+  gap: 3rem;
+  list-style-type: none;
+  justify-content: flex-end;
+  margin-right: 10em;
+}
+
+img {
+  border-radius: 20px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 30px;
+  width: 1100px;
+  height: 600px;
+}
+
+/* Button Section */
+
+.category--bar{
+  display:flex;
+  justify-content: center;
+  padding: 3rem;
+}
+
+button {
+  background:linear-gradient(rgb(18, 214, 240),rgb(23, 241, 151));
+  padding: 10px 30px;
+  margin-right: 50px;
+  border-radius: 200px;
+  background-color: #fff;
+  font-size: 15px;
+  border: none;
+  cursor: pointer;
+  transition: opacity 0.3s;
+}
+
+button:hover {
+  opacity:0.5;
+}
+
+button:focus{
+  background:linear-gradient(cyan,rgb(255, 255, 255));
+}
+
+
+
+a{
+  text-decoration: none;
+  color: white;
+}
+
+.image {
+  width: 500px;
+  height:400px;
+}
+
+.description {
+  padding:10px;
+}
+
+.container--1 h1 {
+  text-align: center;
+  padding-right: 500px;
+}
+
+h1 {
+  font-size: 64px;
+  color:red;
+  padding-top: 3%;
+  margin-left: 100px;
+  
+}
+.container--1,h1 {
+  margin-right: 100px;
+  padding-left: 20%;
+  margin-top:150px;
+}
+
+.container {
+  
+  display: inline-block;
+  justify-content: center;
+  align-items: center;
+  border:solid rgb(49, 214, 255);
+  width: 80%;
+}
+
+/* Gallery 1 */
```

Comment:
When your having this kind of repetitive component use the same class. And if you want them to change separately add those separate changes using pseudo elements like :nth child. Try to learn about them and add it to the next issue.